frontend/src/SearchBar.js

```diff
@@ -92,15 +92,15 @@ class SearchBar extends Component {
         return [
             {
                 label: 'Departments',
-                options: fuzzysort.go(inputValue, autocompleteOptions[0].options, {keys: ['title', 'desc'], threshold: -200, limit: 10}).map((a) => a.obj)
```

Comment:
I made fuzzysort search through the keywords, which is the title and the desc for Departments and Instructors, but the title and all past names for Courses. Should be functionally identical for everything except Courses, where it should be an improvement!